.github/workflows/test.yml

```diff
@@ -50,7 +44,7 @@ jobs:
           cache: true
       - uses: hashicorp/setup-terraform@v3.1.2
         with:
-          terraform_version: ${{ matrix.terraform }}
+          terraform_version: "1.10.*"
```

Comment:
Do we want to do the same thing on line 34?

Reply:
yeah I'm gonna do that

Reply:
Note this was actually invalid input since I don't think the matrix was actually defined at this point.

Reply:
That's what caught my eye on line 34. I spent a few minutes trying to figure out where and in what clever way the matrix was defined until I noticed your fix here